### ISSUE TITLE
Eris doesn't seem to compile in Dream Daemon

### ISSUE THREAD

**Issue by ModDownloading** [discordia-space/CEV-Eris#5813]
After I compile Eris code in Dream Maker and proceed to try and run it in Dream Daemon, the program crashes without giving me an error. I am currently running the most recent version of BYOND on Windows 7, and there aren't any issues with me joining/playing on the official servers. Other codebases I have compiled run fine on Dream Daemon, so I don't think it's a problem with my BYOND or its compiling and running programs.

**Comment by ModDownloading** [comment]
Thanks to Lazyrobot123, the issue has been identified as the BYOND version I was using being 513.1539 instead of 513.1511, and rolling back the version fixed the issue. Still leaving this open since I don't know if the most recent version being compatible to host is supported or not.

**Comment by hyperioo** [comment]
Yeah, I updated my BYOND recently and I had to roll back to 513.1528 to be able to host locally.
